generator/src/main/java/com/linkedin/pegasus/generator/FileFormatDataSchemaParser.java

```diff
@@ -92,8 +92,9 @@ public FileFormatDataSchemaParser(String resolverPath, DataSchemaResolver schema
               while (entries.hasMoreElements())
               {
                 final JarEntry entry = entries.nextElement();
-                if (!entry.isDirectory() && entry.getName().endsWith(_schemaParserFactory.getLanguageExtension()) &&
-                    (entry.getName().startsWith(_schemaResolver.getSchemasDirectoryName().getName()) || isExtensionEntry(entry)))
+                if (!entry.isDirectory() &&
+                    entry.getName().endsWith(_schemaParserFactory.getLanguageExtension()) &&
+                    (entry.getName().toLowerCase().startsWith(_schemaResolver.getSchemasDirectoryName().name().toLowerCase())))
```

Comment:
1. I'm not sure what the original logic (before the `SchemaDirectoryName` change) looked like, but would it make sense to add a `+ "/"` to the check? Otherwise cases like `pegasusFoo/Foo.pdl` would not be filtered out. However, we need to make sure we don't break existing behavior.

2. I'm curious about this bug. Why do we want to allow case-insensitive matches? I'll follow up offline.

Reply:
For #1 , yeah more or less your idea is great, but the original logic is like this
For #2, it is because we add to jar with lower cases, but the directory name defined Resolver is in upper case. That is why we need case insensitive. Follow up with me offline anytime

Reply:
If this is the case, ignoring the case is generally ok.
but why did we add to jar using lower case ?

Reply:
It is one of the original choice I guess

Reply:
When a enum is being used, the method `name()` will return its original form which in our case is all upper cases. On the other hand, the method `getName()` will return the actual coded content literal, which in our is all lower cases.
In the beginning of my very first PR of this parser, I was using `name()` so it is not matching the configured entry prefix in jar, but I have the isExtensionEntry to help to rescue.